docs/lakehouse-iceberg-catalog.md

```diff
@@ -111,10 +111,28 @@ For `bucket` and `truncate`, the first argument must be integer literal, and the
 
 ### Table distributions
 
-- Doesn't support `Distribution`, you should use `BucketPartition` instead.
+- Gravitino used by default `HashDistribution`. Hash distribute by partition key.
+```json
+{
+  "strategy": "hash"
+}
+```
+```java
+Distributions.hash();
+```
+
+- Support `RangeDistribution`, You can pass `range` as values through the API. Range distribute by partition key or sort key if table has an SortOrder.
+```json
+{
+  "strategy": "range"
+}
+```
+```java
+Distributions.range();
+```
 
 :::info
-If you load Iceberg tables, the table distribution strategy is `hash` with num 0, which means no distribution.
+Iceberg automatically calculates the bucket value and performs hashing based on the partition field, or automatically calculates ranges according to SortOrder.
```

Comment:
Why do we mix the concepts of buckets and partitions?

Reply:
Iceberg's distribution shouldn't be bind to bucket.

Reply:
@qqqttt123 any suggestion to the description?

Reply:
Bucket is not a good concept here.  We should avoid it.

Reply:
Em. This is confusing for Iceberg users. Iceberg can't specify distribution expressions.

Reply:
yes, this's a difference between gravitino distribution and Iceberg distribution, so I add it.  could you help to provide a more useful info?